docs/contributing-to-airbyte/building-new-connector/testing-connectors.md

```diff
@@ -50,6 +54,24 @@ These inputs are all described in the [Airbyte Specification](../../architecture
 
 Since the inputs to standard tests are often static, the file-based runner is sufficient for most connectors. However, in some cases, you may need to run pre or post hooks to dynamically create or destroy resources for use in standard tests. For example, if we need to spin up a Redshift cluster to use in the test then tear it down afterwards, we need the ability to run code before and after the tests, as well as customize the Redshift cluster URL we pass to the standard tests. If you have need for this use case, please reach out to us via [Github](https://github.com/airbytehq/airbyte) or [Slack](https://slack.airbyte.io). We currently support it for Java & Python, and other languages can be made available upon request.
 
+## Airbyte's Standard Tests (v2) - Source Acceptance Tests
+
+Unfortunately Standard Tests (v1) has some problems: 
+* Its tests are opaque to a contributor, difficult to debug when something goes wrong
+* It tries to encompass testing all features/edge cases in one test case which has made it difficult to configure and its behavior hard to understand
+
+Source Acceptance Tests tries to address these issues. These are the differences from v1:
+* configured with YAML file, forget about Gradle
+* individual tests can be disabled in the config file
+* individual tests can have different inputs and run multiple time with different sets of inputs
+* each test has a timeout
+* each test has stores all inputs and outputs 
+* use pytest as a test runner
+* can be run manually, with interactive debugger, custom plugins, last failed tests, etc - thanks to pytest
+* the output is easy to read, good bye Java stacktrace - thanks to pytest
```

Comment:
Feels like this is "apologizing" for mistakes in the past but most people will not have used the previous one so we should focus on their experience

```suggestion
```